### PR TITLE
Add support for providing priority class name, solves #380

### LINKED ISSUE #380: Polaris: Support for custom priority class name
**Is your feature request related to a problem? Please describe.**
We should have a way to specify the priority for the pod by means of priorityClassName for polaris-dashboard Kubernetes deployment.

**Describe the solution you'd like**
A property in values.yaml like 'priorityClassName' that can take the name of existing priority class that users have created.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by AshutoshNirkhe**
**Why This PR?**
_a short description of why this PR is needed_

Fixes #380

**Changes**
Changes proposed in this pull request:

*Added support to use priorityClassName in Polaris dashboard deployment.

**Checklist:**

* [ ] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

File: stable/polaris/values.yaml [hunk 1]
```diff
@@ -37,6 +37,8 @@ dashboard:
     limits:
       cpu: 150m
       memory: 512Mi
+  #Priority Class name to be used in deployment if provided.
```
Comment (sudermanjr) [inline stable/polaris/values.yaml]: Linter:
```suggestion
  # Priority Class name to be used in deployment if provided.
```

File: stable/polaris/templates/dashboard.deployment.yaml [hunk 2]
```diff
@@ -31,6 +31,9 @@ spec:
         configMap:
           name: {{ include "polaris.fullname" $ }}
       {{- end }}
+      {{- if .Values.dashboard.priorityClassName }}
+      priorityClassName: {{ .Values.dashboard.priorityClassName | quote }}
```
Comment (rbren) [inline stable/polaris/templates/dashboard.deployment.yaml:35]: should we add this to the webhook as well?

**Review by rbren — APPROVED** [review]

**Review by baderbuddy — COMMENTED** [review]
This looks great. Thanks for the contribution! Would you mind adding these 2 new values to the README?

**Comment by AshutoshNirkhe** [comment]
> This looks great. Thanks for the contribution! Would you mind adding these 2 new values to the README?

Thanks @baderbuddy sure thing, updated.

**Review by baderbuddy — APPROVED** [review]
LGTM

**Pull request merged**